src/neuroglancer/datasource/dvid/frontend.ts

```diff
@@ -90,21 +101,38 @@ export class VolumeDataInstanceInfo extends DataInstanceInfo {
     this.numChannels = 1;
   }
 
-  getSources(chunkManager: ChunkManager, parameters: VolumeChunkSourceParameters) {
-    return [Array
-                .from(VolumeChunkSpecification.getDefaults({
-                  voxelSize: this.voxelSize,
-                  dataType: this.dataType,
-                  numChannels: this.numChannels,
-                  lowerVoxelBound: this.lowerVoxelBound,
-                  upperVoxelBound: this.upperVoxelBound,
-                  volumeType: this.volumeType,
-                }))
-                .map(spec => {
-                  return chunkManager.getChunkSource(
-                      VolumeChunkSource, stableStringify(parameters),
-                      () => new VolumeChunkSource(chunkManager, spec, parameters));
-                })];
+  getSources(chunkManager: ChunkManager, parameters: DVIDSourceParameters) {
+    let sources: VolumeChunkSource[][] = [];
+    for (let level = 0; level < this.numLevels; ++level) {
+      let voxelSize = vec3.clone(this.voxelSize);
```

Comment:
I think it would still make sense to call VolumeChunkSpecification.getDefaults, since that automatically enables client-side transcoding to compressed segmentation format, and in the future might intelligently pick the chunkDataSize based on the voxel size.